src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-NODE_VERSION_DEFAULT = "8.11"
+NODE_VERSION_DEFAULT = "10.14"
```

Comment:
@yiliaomsft & @ahmedelnably should the function app create use the new Default Version for Node on Linux as well? Currently it hard-coded to use 8.11.1. Thanks!

Reply:
CC: @ankitkumarr can you follow up Ahmed & check if you need this change for functionapp for linux & handle it accordingly, currently functionapp create is setting 8.11.1 as default version for both Linux & windows.

Reply:
Yeah we should now default to 10.

Reply:
Done updated this to 10.14.1